apps/velo-external-db/test/e2e/app_data.e2e.spec.ts

```diff
@@ -144,12 +140,12 @@ describe(`Velo External DB Data REST API: ${currentDbImplementationName()}`,  ()
     testIfSupportedOperationsIncludes(supportedOperations, [ DeleteImmediately ])('bulk delete api', async() => {
         await schema.givenCollection(ctx.collectionName, [ctx.column], authOwner)
         await data.givenItems(ctx.items, ctx.collectionName, authAdmin)
-await data.givenItems([ ctx.items[1] ], ctx.collectionName, authAdmin)
+        await data.givenItems([ ctx.items[1] ], ctx.collectionName, authAdmin)
         const response = await axiosInstance.post('/items/remove', { 
             collectionId: ctx.collectionName, itemIds: ctx.items.map(i => i._id) 
         }, authAdmin)
 
-        expect(response.data.results).toEqual(expect.toIncludeSameMembers(ctx.items))
+        expect(response.data.results).toEqual(expect.toIncludeSameMembers(ctx.items.map(dataConvertUtils.asWixDataItem)))
```

Comment:
I prefer not using here asWixData, because IMO that's part of what we are checking. (if someone would change the `asWixData` function to be without {item: ...} it would pass.
lets keep the results here raw, or with specific matchers, but without using these functions